config/routes.yaml

```diff
@@ -209,3 +209,7 @@ status:
     methods: [GET]
     controller: null
 
+api_cities.json:
+    path: /api/v1/cities.json
+    methods: [GET]
```

Comment:
Maybe we should change the endpoint to POST to avoid random crawlers triggering empty database queries